### DISCUSSION TITLE
missing gunners in hoplitas vehicles (CUP)

### DISCUSSION THREAD

**Discussion by STyx2909** [Vdauphin/HeartsAndMinds#1650] [Question and answer]
I am sure this has been covered many time but can not find the info.
What is the cause for missing gunners in hoplitas vehicles (CUP).
They were there at first but now are missing every session.

**Comment by STyx2909** [comment]
Here is the server log.
[arma3server_2024-08-31_23-37-25.txt](https://github.com/user-attachments/files/16935741/arma3server_2024-08-31_23-37-25.txt)

Have not found anything in it related.

**Comment by Vdauphin** [comment]
Hello,

> I am sure this has been covered many time but can not find the info.

No

Working from here:
![20240911005444_1](https://github.com/user-attachments/assets/1532c200-7bcb-4b1c-b9c1-aa3afb0d7fbf)
![20240911005520_1](https://github.com/user-attachments/assets/a65db3eb-33ff-452f-a1c3-d068f02cbae5)

Please try with only ACE and CUP

Cheers

**Reply by STyx2909** [reply]
It has been a long time since I posted this, and we stalled the campaign somewhat.

I was looking to port our custom campaign to North Takistan instead, for some scenery change. I stumbled upon the issue again. Local MP server.

because of the changes we made into core/def/mission.sqf and params I tried these on a fresh Altis built without any troubles. Every vehicles are spawning with crew, while in Takistan and North Takistan, many times, crew is missing. Not always, but often.

So, I thought you might be interested in testing these.

MOD list, ACE + CUP + CFP
[def.zip](https://github.com/user-attachments/files/20196319/def.zip)

**Reply by STyx2909** [reply]
Forget this. I had other MODs loaded as well.
Was tired. LOL.
